core/kits/assets/css/panel.scss

```diff
@@ -1,3 +1,5 @@
+$editor-kit-accent: #4ab7f4;
```

Comment:
It should be moved to the editor's variables file

Reply:
It's a private var